ui/widgets/autocomplete.js

```diff
@@ -539,9 +542,10 @@ $.widget( "ui.autocomplete", {
 		// Size and position menu
 		ul.show();
 		this._resizeMenu();
+		var pos = this.isRtl() ? { my: "right top", at: "right bottom", collision: "none" } : this.options.position;
```

Comment:
Should I create `_getPositionRtl: function( position )` which replace every `left` with `right` & vice versa when `this.isRtl() == true` ?
If so, should I put it in widget.js as it might be used within others widgets ?